colab/static/js/subscriptions.js

```diff
@@ -0,0 +1,32 @@
+$(function (){
+
+  ENTER_BUTTON = 13;
+  searchMaillingLists();
```

Comment:
Yep, this was the main intent of this line.. Glad the test is passing again now!